code/modules/networks/computer3/buildandrepair.dm

```diff
@@ -2,7 +2,7 @@
 /obj/item/motherboard
 	name = "Computer mainboard"
 	desc = "A computer motherboard."
-	icon = 'icons/obj/module.dmi'
+	icon = 'icons/obj/machinery/module.dmi'
```

Comment:
I don't care as much about this, but this seems like another thing that isn't machinery.